angular/src/app/home/home.component.scss

```diff
@@ -33,34 +18,58 @@
     padding-top: 10px;
     font-size: 0.7em;
   }
-
-  .sizeImg {
-    max-width:75%;
-  }
-
 }
 
-.card-over {
-  margin-top: -75px;
-}
+.home-card {
+  margin: -50px 10px 10px 10px;
+  padding: 5px;
 
-.home-title {
-  padding-bottom: 25%;
-  padding-top: 5%;
-  background-image: url('../../assets/images/background-dish.png'), url('../../assets/images/star.png'), url('../../assets/images/Wood2.jpg');
-  background-position: bottom, top, center;
-  background-size: cover, 15%, cover;
-  background-repeat: no-repeat, no-repeat, repeat;
-}
+  .tiles {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+
+    .tile-content {
+      padding: 5px;
+      text-align: center;
 
-@media (max-width: 600px) {
-  .forDesktop {
-    display: none;
+      h3 {
+        text-transform: uppercase;
+      }
+    }
+    .tile-action {
+      display: flex;
+      justify-content: center;
+      button {
+        text-transform: uppercase;
+      }
+    }
   }
 }
 
 @media (min-width: 600px) {
-  .forMobile {
-    display: none;
+  .home-header {
+    .title {
+      font-size: 3em;
+    }
+
+    .subtitle {
+      font-size: 1em;
+    }
+  }
+
+  .home-card {
+    margin-top: -75px;
+    margin-left: 48px;
+    margin-right: 48px;
+    padding: 16px;
+
+    .tiles {
+      flex-direction: row;
+
+      tile-content {
```

Comment:
I guess that this should be `.tile-content` right?

Reply:
yes :D obviously it should :P